### PR TITLE
Code for scrollbar styling added

### LINKED ISSUE #82: The scrollbar is not very attractive
### Description of the enhancement. ✍🏻

The website is beautiful but the scrollbar is very simple.
I can add styling to the scrollbar so that it looks more attractive and the website also looks better.
I can change the styling of scrollbar .Can you assign this issue to me?

### Describe briefly how you would implement this enhancement.

```bash
Some steps to implement this enhancement-
1. Step 1-I will change the color of scrollbar 
2. Step 2-Adjust its width
3. Step 3-Change its color on hovering
```

### Screenshots Example (if applicable)

_No response_

### Additional information

_No response_

### Contributor terms

- [X] I agree to follow the Contributing Instructions

### PR DESCRIPTION
**Pull request by Shreyans2004**
## Related Issue(s)

This pr is related to pr#88.I have resolved the issues 
The code for scrollbar is added.
Can it be merged now?
Closes #82
## Type of Change

- [ ] New Feature (Landing Page)
- [ ] New Feature (App)
- [ ] Bug (Landing Page)
- [ ] Bug (App)
- [ ] New Component (App)
- [ ] Enhancement
- [ ] Documentation / ReadMe / Workflows
- Other (If not in above options):  

## Your Idea for Level (GSSoC)

- [ ] Level 1
- [x] Level 2
- [ ] Level 3

### CODE REVIEW

File: src/styles/App.scss [hunk 1]
```diff
@@ -9,4 +9,23 @@
...
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: white; 
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: orange; 
+}
+
```
Comment (praveenscience) [inline src/styles/App.scss:30]: Please format this... 😊

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: src/styles/App.scss (hunk 1 above)
Comment (Shreyans2004) [reply src/styles/App.scss]: Ya sure

**Comment by Shreyans2004** [comment]
I have formatted the file. Can it be merged now?

File: src/styles/App.scss [hunk 2]
```diff
@@ -10,22 +10,21 @@
...
-  background: black;
-}
- 
-/* Handle */
-::-webkit-scrollbar-thumb {
-  background: white; 
-}
+  /* Track */
+  ::-webkit-scrollbar-track {
+    background: black;
```
Comment (praveenscience) [inline src/styles/App.scss:18]: Make this into `#000`.

File: src/styles/App.scss [hunk 3]
```diff
@@ -10,22 +10,21 @@
...
+  }
 
-/* Handle on hover */
-::-webkit-scrollbar-thumb:hover {
-  background: orange; 
-}
+  /* Handle */
+  ::-webkit-scrollbar-thumb {
+    background: white;
```
Comment (praveenscience) [inline src/styles/App.scss:23]: Make this into `#fff`.

File: src/styles/App.scss [hunk 4]
```diff
@@ -10,22 +10,21 @@
...
+  }
 
+  /* Handle on hover */
+  ::-webkit-scrollbar-thumb:hover {
+    background: orange;
```
Comment (praveenscience) [inline src/styles/App.scss:28]: Make this into `#ffa500`.

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by AmanJain18** [comment]
@Shreyans2004 Why two PR make it into one and open pr on respected branch. Not on Maain Branch.

**Comment by AmanJain18** [comment]
@praveenscience Sir Don't merge this, as pr is not on correct branch.
Changes done in website so pr should be open targetting `website` branch not `main` branch.

**Comment by praveenscience** [comment]
Okay, since this is not what's expected, scrollbar styling is for the main landing page, let's close this.

**Pull request closed without merging**